modules/ROOT/pages/raise-error-component-reference.adoc

```diff
@@ -4,23 +4,90 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: mule, esb, studio, raise, error
 
-This Core component generates a Mule Error, as if a failure had occurred, which allows you to customize its description and type.
+This core component generates a Mule error, as if a failure had occurred, which allows you to customize its description and type.
 
-Use this component to only raise core runtime errors, such as `MULE:SECURITY`, `MULE:CONNECTIVITY`, etc, or custom ones.
+Use this component to only raise:
+
+* Core runtime errors, such as `MULE:SECURITY`, `MULE:CONNECTIVITY`, etc.
+* Custom error types. +
+
+[NOTE]
 You cannot raise an error from another connector or module like `FILE:FILE_NOT_FOUND`, `JSON:INVALID_SCHEMA`. You cannot use a connector's existing namespace.
 
-For core runtime errors types, you must use the implicit namespace and identifier, you can only customize the error's description message. +
-For custom error types, declare a new namespace. The namespace of an error type should help you identify the origin of an error. +
+== Raise Error Configuration
+
+[%header,cols="1,1,2,2"]
+|===
+| Field | Value | Description | Example
+
+| Description | String or DataWeave expression | Specifies the error description. |
+`description="Invalid email. Cannot complete transaction."`
+
+| Type | String | Specifies the type of the error. |
+`type="ORDER:INVALID_DATA"`
+
+|===
+
+== Raise Core Runtime Errors Types
+For core runtime errors types, you must use the implicit namespace and identifier, you can only customize the error's description message. For example:
+
+[source,xml,linenums]
+----
+<raise-error type="MULE:CONNECTIVITY" description="Error description message"/>
+----
+
+== Raise Custom Errors Types
+For custom error types, declare a new namespace. The namespace of an error type should help you identify the origin of an error. For example:
+
+[source,xml,linenums]
+----
+<raise-error type="CUSTOM:CUSTOM_ERROR" description="Error description message"/>
```

Comment:
I would swap this example of a `type` with the one in the configuration. I think it makes more sense to give a generic example in the configuration explanation and then use a specific one like the email thing here, specially since you are mentioning the fact that a namespace should help identify the origin.